src/containers/NavigationContainer.ts

```diff
@@ -12,7 +12,7 @@ const mapStateToProps = (state: KialiAppState) => ({
 });
 
 const mapDispatchToProps = (dispatch: ThunkDispatch<KialiAppState, void, KialiAppAction>) => ({
-  setNavCollapsed: (collapse: boolean) => dispatch(UserSettingsThunkActions.setNavCollapsed(!collapse))
+  setNavCollapsed: (collapse: boolean) => dispatch(UserSettingsThunkActions.setNavCollapsed(collapse))
```

Comment:
I comment this just in case bc I don't know the rationale behind that action. is that negative operator (!) intended?

Reply:
Yes, the component change and not We not need set this to negative, in the pf3 component was navCollapsed and now is onOpen so is the opposite logical ^^